internal/pkg/template/workload_test.go

```diff
@@ -290,3 +290,23 @@ func TestRuntimePlatformOpts_IsDefault(t *testing.T) {
 		})
 	}
 }
+
+func TestSsmOrSecretARN_RequiresSub(t *testing.T) {
+	require.False(t, ssmOrSecretARN{}.RequiresSub(), "SSM Parameter Store or secret ARNs do not require !Sub")
+}
+
+func TestSsmOrSecretARN_ValueFrom(t *testing.T) {
+	require.Equal(t, "/github/token", SecretFromSSMOrARN("/github/token").ValueFrom())
+}
+
+func TestSecretsManagerName_RequiresSub(t *testing.T) {
+	require.True(t, secretsManagerName{}.RequiresSub(), "secrets refering a SecretsManager name need to be expanded to a full ARN")
```

Comment:
tiniest of tiny nits: 
```suggestion
	require.True(t, secretsManagerName{}.RequiresSub(), "secrets referring to a SecretsManager name need to be expanded to a full ARN")
```